README.md

```diff
@@ -1,2 +1,55 @@
 # emqx-operator
+
 A Kubernetes Operator for EMQ X Broker
+
+**Project status: *alpha*** Not all planned features are completed. The API, sepc, status and other user facing objects may chage, but in a backward compatible way.
+
+Note: Project was previously known as emqx/emqx-operator.
+
+## Overview
+
+The EMQ X Operator provieds [Kubernetes](https://kubernetes.io/) native deployment and management of [EMQ X](https://www.emqx.io/). The purpose of this project is to simplify and automate the configuration of EMQ X Broker for EMQ X cluster.
+
+The EMQ X Operator includes, but is not limited to, the following features:
+
+* **Kubernetes Custom Resource**: Use Kubernetes custom resource to deploy and manage EMQ X Broker.
```

Comment:
Use Kubernetes custom resource to deploy and manage EMQ X Broker. -> Deploy and manage EMQ X Cluster with pre-defined custom resources